src/theme.c

```diff
@@ -94,16 +109,34 @@ load_buttons(struct theme *theme)
 	};
 
 	char filename[4096] = {0};
+#if HAVE_RSVG
+	int size = theme->title_height - 2 * theme->padding_height;
+#endif
```

Comment:
Should we move the size down into the 2nd `HAVE_RSVG` block?

Reply:
Have done. 0.000001% less efficient, but much clearer and cleaner.